browse-scaler/src/util.js

```diff
@@ -69,13 +72,13 @@ exports.slurpImageIntoBuffer = async imageUrl => {
  * const fs = require('fs/promises')
  * const buffer = await fs.readFile('<filename>');
  */
-exports.readFile = async (f) => {
-  return new Promise ((resolve, reject) => {
-    fs.readFile (f, (err, data) => {
+exports.readFile = async f => {
+  return new Promise((resolve, reject) => {
+    fs.readFile(f, (err, data) => {
       if (err) {
-        reject (err);
+        reject(err);
       }
-      resolve (data);
+      resolve(data);
     });
   });
-}
+};
```

Comment:
Could this be removed and substitute in the built-in `fs.promises.readFile`?

Reply:
I have swapped it. It is the same thing and it cleans things up thank you!